client/src/command/mod.rs

```diff
@@ -0,0 +1,42 @@
+mod health;
+mod node;
+
+use clap::{Arg, Command};
+
+use crate::command::health::HEALTH_CHECK_ENDPOINT_KEY;
+use crate::command::node::NODE_CONFIG_KEY;
+
+pub use health::health_check;
+pub use node::start_node;
+
+pub const NODE: &str = "node";
+pub const HEALTH_CHECK: &str = "healthcheck";
+
+pub fn cli() -> Command {
+    Command::new("ipc-client")
+        .about("The one client to interact with IPC actors")
+        .subcommand_required(true)
+        .arg_required_else_help(true)
+        .allow_external_subcommands(true)
+        .subcommand(
+            Command::new(HEALTH_CHECK)
```

Comment:
Linked to my comments above, we should figure out a cleaner way to pass the subcommands and the handlers to the clients and servers.